src/components/grid/grid.vue

```diff
@@ -221,6 +250,9 @@
       this.$forceUpdate();
     },
     methods: {
+      isRenderer(column) {
+        return column.render && column.render.use;
+      },
       getColumnName(field) {
         return this.columns.find(column => column.field === field).caption;
```

Comment:
이거 조건에 안걸려서 `undefined` 나오면 터져요

Reply:
```suggestion
        const matchedField =  this.columns.find(column => column.field === field);
        return matchedField && matchedField || '';
```

`default` 로 넘길 값은 저 상황에 맞는 값으로 대체하는게 좋을 것 같습니다
현재는 `default` 로 가는 경우는 없고 `throw new Error` 로 떨어질테니 우선 빈 스트링으로

Reply:
`@babel/plugin-proposal-optional-chaining`, `@babel/plugin-proposal-nullish-coalescing-operator` 을 `.babelrc` 플러그인에 넣어주면 아래처럼도 가능합니다.

```js
return this.columns.find(column => column.field === field)?.caption ?? '';
```
사용법도 간단하므로 프로젝트에 어차피 바벨이 설치되어 있으니 고려해볼만한 스펙이라고 생각합니다

Reply:
네 좋은 대안이네요   
일단은 내부에서 사용을 안하니 제거를 하는게 나을거 같네요.  
초창기에 쓸려고 만들었는데 현시점에선 사용을 안하네요